tasties_app/models.py

```diff
@@ -13,6 +13,23 @@
 class Category(models.Model):
     category_name = models.CharField(max_length=16, unique=True)
 
+    def __str__(self):
+        return self.category_name
+
+    @staticmethod
+    def get_recipes_by_category(category):
+        """Returns a QuerySet of Recipes tagged with this Category
+
+        Args:
+            category (Category): Category object
+
+        Returns:
+            QuerySet: QuerySet of Recipes containing given Category
+        """
```

Comment:
Please put the comment one line under the """
just like we do here: https://github.com/redhat-beyond/tasties/blob/19fe6152e13ef9dcbced96747f626ec035c10eac/tasties_app/models.py#L58